android/sourcemaps.gradle

```diff
@@ -14,9 +14,22 @@ gradle.projectsEvaluated {
 task uploadSourcemaps() {
     group 'instabug'
     description 'Uploads sourcemaps file to Instabug server'
-    enabled rootProject.hasProperty('instabugUploadEnable')
-            ? new Boolean(rootProject.property('instabugUploadEnable'))
-            : true
+    enabled = {
+        def envValue = System.getenv('INSTABUG_SOURCEMAPS_UPLOAD_DISABLE')?.toBoolean()
+        def defaultValue = true
+       
+        if(rootProject.hasProperty('instabugUploadEnable')){
+            project.logger.warn "The property instabugUploadEnable is deprecated and will be removed in an upcoming version. \n" +
+            "You can use INSTABUG_SOURCEMAPS_UPLOAD_DISABLE environment variable instead."
+            defaultValue = rootProject.property('instabugUploadEnable')
+        }
+        if(envValue != null){
+            return !envValue 
+        }
+        else {
+            return defaultValue
+        }
+    }()
```

Comment:
I think it would be cleaner if we extract this logic into a function and then use it here.
Let's also make sure the code is properly formatted.